perfact/zodbsync/commands/watch.py

```diff
@@ -311,7 +317,12 @@ def _update_children(self, oid):
                     )
                 )
                 os.makedirs(self.base_dir+newpath)
-                os.rename(self.base_dir+oldpath, self.base_dir+newpath)
+                try:
+                    os.rename(self.base_dir+oldpath, self.base_dir+newpath)
+                except OSError as err:
+                    if err.errno == 2:  # no such file or directory
+                        raise TreeOutdatedException()
```

Comment:
Please add at least a log line, even if it is not an ERROR level message but, for example, INFO.